regression/pages/studio/utils.py

```diff
@@ -0,0 +1,18 @@
+"""
+Utility functions for studio page objects.
+"""
+from opaque_keys.edx.locator import CourseLocator
+
+
+def get_course_key(course_info, module_store='split'):
```

Comment:
@benpatterson I am thinking of getting module_store info from environment variable instead of passing it as an argument.  What are your suggestions?